qcenginerecords/molpro/ccsd_water_energy/output.json

```diff
@@ -44,18 +46,14 @@
     "routine": "qcelemental.models.results"
   },
   "success": true,
-  "return_result": -74.7835019973896,
+  "return_result": -75.0196418084857,
```

Comment:
Good to know even in sto-3g the energy is `>= -75`, can be a quick check from now on.